src/services/api.service.js

```diff
@@ -1,8 +1,14 @@
 const axios = require('axios');
+const { Storage } = require('@google-cloud/storage');
+const path = require('path');
 
 class ApiService {
   constructor(apiUrl) {
     this.apiUrl = apiUrl;
+    this.gc = new Storage({
+      keyFilename: path.join(__dirname, `../api/${process.env.GOOGLE_PROJECT_ACCOUNT_FILE}`),
```

Comment:
You pass the `apiUrl` variable as a param so why you don't pass GOOGLE env variables as params too?

Reply:
Yes, i see. Thank you!